src/v2/view-builder/views/OTPEmailView.js

```diff
@@ -1,14 +1,74 @@
-import { loc } from 'okta';
+import { loc, View, createCallout, createButton, _ } from 'okta';
 import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
+import email from '../shared/email';
+import polling from '../shared/polling';
 
-const Body = BaseForm.extend({
-  title () {
-    return `Email link (${this.options.appState.get('factorEmail')})`;
+const SHOW_RESEND_TIMEOUT = 600;
```

Comment:
How did we come up with this value ? Currently this was set to `const SHOW_RESEND_TIMEOUT = 60000;` In the factorPollVerificationEmailView.

Reply:
this shall be `60000`. modified for testing but forgot to change it back.